lib/logstash/outputs/syslog.rb

```diff
@@ -223,6 +229,8 @@ def connect
     socket
   end
 
+  CRL_END_TAG = "\n-----END X509 CRL-----\n"
```

Comment:
This constant definition has to be moved on top of the file, close to the *_LABEL constants

Reply:
Done!